tracing/tracing.go

```diff
@@ -0,0 +1,54 @@
+// The tracing package provides utilities for the Kiali server
+// to instrument itself with tracing to provide better insights
+// into server performance. Currently only integrated with Jaeger.
+package tracing
```

Comment:
Naming is hard :) and no good answers at all, but I find a little bit confusing to have a "tracing" package given the context that we deal with external services prom and jaeger as both sources and also used for instrumentation.

I saw a comment in that way related the configuration, perhaps, the internal observability aspects may go to a single location (observability, or synced with the name you choose there).

Reply:
I was a little unsure of where to put this package or name it when I created it. There's already a `jaeger` package for querying jaeger directly and a `prometheus` package for the same. Since this mostly adds just opentelemetry tracing, there is very little jaeger specific code. `observability` sounds fine to me. Internal metrics/logs integration with otel can be put there if added in the future.